rinex/src/navigation/health/gps.rs

```diff
@@ -1,86 +1,140 @@
-//! GPS Sv Health specifications
+//! GPS SV Health specifications
 use bitflags::bitflags;
 
-/// GPS Sv Health indication
+/// GPS SV Health indication
 #[derive(Debug, Clone, PartialEq, PartialOrd)]
 #[cfg_attr(feature = "serde", derive(Serialize))]
 pub enum GPSHealth {
-    /// GPS Legacy Sv Health indication
-    LNAV(LegacyHealth),
-    /// GPS CNAV Sv Health indication
-    CNAV(CivilianHealth),
-    /// GPS CNV2 Sv Health indication
-    CNV2(Civilian2Health),
+    /// GPS Legacy SV Health indication
+    LNAV(LNAVHealth),
```

Comment:
I think having the names of the structs similar to the enum cases is less confusing